projects/vc-api-implementation.md

```diff
@@ -0,0 +1,240 @@
+
+Furthermore, Energy Web has documentation on VC-API and general documentation on SSI available the Energy Web gitbook: 
+
+- https://energy-web-foundation.gitbook.io/energy-web/foundational-concepts/self-sovereign-identity-introduction
+- https://energy-web-foundation.gitbook.io/energy-web/ew-dos-technology-components-2023/identity-and-access-management-iam/iam-guides/verifiable-credential-api
+
+# Alignment with the OpenWallet Foundation Mission
+_Include a statement on alignment with the [OpenWallet Foundation mission](https://openwallet-foundation.github.io/tac/governance/charter/)._
+
+The project team believes that the project is well aligned with the OpenWallet Foundation mission. 
+
+An implementation of VC API developed and hosted by the OWF would advance the adoption of interoperable and privacy enhancing digital wallets. This is because VC API is an open specification of an interface for Verifiable Credential operations, which are relevant functions for digital wallets.
+Furthermore, as an HTTP API, the project is a technology well understood by many organizations and will plausibly accelerate their efforts to develop or integrate digital wallets. 
+
+In addition, development of a VC API implementation would promote the development and proliferation of the VC API specification itself, which is an open standard related to digital wallets. 
+
+The project has been open source since its inception and aims to be inclusive, in so far as it should be usable across a wide range of domains and infrastructure requirements.
+
+# Current Code of Conduct
+_Has this project adopted a Code of Conduct? If so, please include a link to it._
+
+https://github.com/energywebfoundation/ssi/blob/develop/apps/vc-api/CODE_OF_CONDUCT.md
+
+# TAC Sponsor
+_Include the name of a sponsor from the TAC, if identified (a sponsor helps mentor projects)_
+
+The project has not yet identified a TAC sponsor.
+
+# Project License
+_Include the proposed project license (Apache 2.0 by default)._
+
+GNU General Public License 3.0 -https://github.com/energywebfoundation/ssi/blob/develop/apps/vc-api/LICENSE 
+
+# Source Control
+_If the project already exists, please provide a link to the repositories._
+
+The project source code is at the following location: https://github.com/energywebfoundation/ssi/tree/develop/apps/vc-api 
+
+The git repository containing the code is a monorepo which contains the applications and libraries in the tables below, in addition to the VC API implementation. *To enable the donation of the project to OWF, Energy Web and Elia are willing to refactor the project as required and have the development resources for doing so.* For example, the monorepo could be converted to a simple npm project, with the libraries subsumed in the VC API implementation and the “input-descriptor-to-credential" application separated into a new project and repository. 
```

Comment:
Could we prepare a list of all the single repositories the `vc-api` needs and will be part of this proposal? 

I would also suggest supplementing this list with the purpose of these additional repos and their relationship/dependencies to `vc-api`. 

For licensing purposes, will we leave related repositories in the organization they currently are? Should we expect a licensing conflict in this case?

Reply:
Hi @skounis , all of the code to be donated is within the `ssi` monorepo repository. The [/vc-api](https://github.com/energywebfoundation/ssi/tree/develop/apps/vc-api) path is the main code to be donated. In addition to this, the two "libraries" mentioned [here](https://github.com/energywebfoundation/owf-project-proposals/blob/vc-api-proposal/projects/vc-api-implementation.md#monorepo-libraries), which are within the monorepo, would be required as well.

No other code from Energy Web or Elia repositories is required. Key external dependencies are listed in the proposal [here](https://github.com/energywebfoundation/owf-project-proposals/blob/vc-api-proposal/projects/vc-api-implementation.md#external-dependencies).

Reply:
@tkuhrt @skounis Do you know if the OWF has any guidance on how repositories should be structured, in particular on whether or not javascript monorepos are recommended? Do you have any personal preference on the matter?
I am asking as I believe that the `vc-api` application should be isolated from the other application in the `ssi` repository, so that it can be donated independently to the OWF. 
My recommendation at this point is to remove all code not necessary to vc-api from the [ssi repository](https://github.com/energywebfoundation/ssi) and then donate the repository as is (with the monorepo framework still in place) after other necessary tasks (such as relicensing & DCO) have been completed.